src/Networks/Rutanio/Rutanio/Networks/Policies/RutanioStandardScriptsRegistry.cs

```diff
@@ -0,0 +1,62 @@
+using System.Collections.Generic;
+using System.Linq;
+using NBitcoin;
+using NBitcoin.BitcoinCore;
+
+namespace Rutanio.Networks.Policies
+{
+    /// <summary>
+    /// Blockcore sample coin-specific standard transaction definitions.
+    /// </summary>
+    public class RutanioStandardScriptsRegistry : StandardScriptsRegistry
+    {
+        // See MAX_OP_RETURN_RELAY in Rutanio-Qt, <script.h>
+        public const int MaxOpReturnRelay = 40;
```

Comment:
If you want to make this param bigger now is your chance as blockcore nodes will become more widely  used (look at BTC size and use that I recommend as it can hold two keys which is needed for P2WSH staking)

Reply:
Thanks for the recommendation. We have changed it to hold two keys (83)